### PR TITLE
Update WinEventParser #5

### PR DESCRIPTION
**Pull request by z0ne323**

### CODE REVIEW

File: WinEventParser/main.py [hunk 1]
```diff
@@ -140,6 +141,17 @@ def get_time_back_in_local(utc_time):
     return local_time
 
 
+def load_rules_list_json(json_event_file):
```
Comment (nigleweber) [inline WinEventParser/main.py:144]: Nice function!

File: WinEventParser/AnalysisRules.json [hunk 2]
```diff
@@ -1,56 +1,316 @@
...
+            "required_count": 3,
+            "window": "60",
+            "action": "alert",
+            "description": "Special privileges were assigned to a new logon. This could indicate a security vulnerability or an attempted privilege escalation."
+        },
+        {
+            "Type": "count",
+            "ID": 4688,
+            "required_count": 3,
+            "window": "60",
```
Comment (nigleweber) [inline WinEventParser/AnalysisRules.json:223]: Is this 60 seconds or 60 milliseconds?

File: WinEventParser/AnalysisRules.json [hunk 3]
```diff
@@ -1,56 +1,316 @@
...
+            "action": "log",
+            "description": "An account was successfully logged on. This is a normal event and should be logged for auditing purposes."
+        },
+        {
+            "Type": "count",
+            "ID": 4625,
+            "required_count": 5,
+            "window": "60",
```
Comment (nigleweber) [inline WinEventParser/AnalysisRules.json:207]: Is this minutes or milliseconds?

**Comment by nigleweber** [comment]
Looking good. @v3r1t4s you did a great job adding so many Event IDs and rules to the project since last week! 🏅 

A couple of quick improvements we can make:
1. For unordered rules we can make make the ID and IDs field the same. We'll need to add code for detection + handling of ID lists elsewhere in the code though. This gives us the option of not needing to specify an explicit "first" event. <- We'll discuss the implications tomorrow aka New Rule Type or explanation of the existing unordered rules in comments
2. The Window field in the rules does not specify what time unit it targets. Presumably this should be milliseconds since that's what timestamps are in. 
  - Add references to what time unit the Window uses
  - Update rules as appropriate so that they have the correct value eg. 60 second should have a value of `60000`

Note:
Rename existing `unordered` to `match_first_unordered`. `match_first_unordered` must match event X first then all of set Y.

File: WinEventParser/AnalysisRules.json (hunk 2 above)
Comment (z0ne323) [reply WinEventParser/AnalysisRules.json:223]: 60 milliseconds

File: WinEventParser/AnalysisRules.json (hunk 3 above)
Comment (z0ne323) [reply WinEventParser/AnalysisRules.json:207]: milliseconds

**Comment by z0ne323** [comment]
> Looking good. @v3r1t4s you did a great job adding so many Event IDs and rules to the project since last week! 🏅
> 
> A couple of quick improvements we can make:
> 
> 1. For unordered rules we can make make the ID and IDs field the same. We'll need to add code for detection + handling of ID lists elsewhere in the code though. This gives us the option of not needing to specify an explicit "first" event. <- We'll discuss the implications tomorrow aka New Rule Type or explanation of the existing unordered rules in comments
> 2. The Window field in the rules does not specify what time unit it targets. Presumably this should be milliseconds since that's what timestamps are in.
> 
> * Add references to what time unit the Window uses
> * Update rules as appropriate so that they have the correct value eg. 60 second should have a value of `60000`

On it as I speak!

**Pull request closed without merging**